src/client/components/Modes/Modes.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import '../../styles/common.css';
+import ModesItem from '../ModesItems/modesItem';
+import { Image3d, ImageHeadSet, ImageSpeaker, ImageExtra } from 'src/images';
+
+class Modes extends Component {
+
+  render() {
+    return (
+      <div className="modes-container">
+        <h3 className="modes-title">Modes</h3>
+        <div className="modes">
+          <ModesItem iconUrl={Image3d} text='3D' />
```

Comment:
For this case you can create an array of images and map through them to reduce code (ask me if you have any questions)